apps/src/weblab2/layout/VerticalLayout.tsx

```diff
@@ -57,17 +96,22 @@ const VerticalLayout: React.FunctionComponent<LayoutProps> = ({
               style={{width: leftPanelWidth}}
               className={moduleStyles.flexShrink0}
             />
-            <ResizeBar
-              isVertical={true}
-              separatorProps={leftPanelSeparatorProps}
-              isDragging={leftPanelDragging}
-            />
+            {!shouldHideEditor && (
+              <ResizeBar
```

Comment:
The `ResizeBar` acts strangely next to the HTML preview so remove in widget view.